src/app/tasks/checker-inbox-and-tasks-tabs/checker-inbox/checker-inbox.component.html

```
@@ -66,8 +65,8 @@
             }
           </mat-select>
         </mat-form-field>
-        <mat-form-field class="flex-31">
-          <mat-label>{{ 'labels.inputs.Entity' | translate }}</mat-label>
+        <mat-form-field style="width: 280px">
```

HTMLHint (1.8.0)

[warning] Inline style [  style="width: 280px" ] cannot be used.

(inline-style-disabled)